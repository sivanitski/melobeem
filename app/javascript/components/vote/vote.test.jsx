import React from "react";
import { BrowserRouter as Router } from "react-router-dom";
import renderer from "react-test-renderer";

import Vote from "./vote";

const mockMatch = {
  match: {
    params: {
      id: "111",
    },
  },
};
describe("Components", () => {
  describe("Vote", () => {
    it("should render Vote component", () => {
      const component = renderer
        .create(
          <Router>
            <Vote match={mockMatch} />
          </Router>
        )
        .toJSON();

      expect(component).toMatchSnapshot();
    });
  });
});
