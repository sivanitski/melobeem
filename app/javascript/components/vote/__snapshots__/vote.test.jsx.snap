// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components Vote should render Vote component 1`] = `
<div
  className="loading"
>
  Loading...
</div>
`;
